modules/builds-adding-input-secrets-configmaps.adoc

```diff
@@ -51,7 +51,7 @@ data:
 [source,terminal]
 ----
 $ oc create secret generic secret-mvn \
-    --from-file=id_rsa=<path/to/.ssh/id_rsa>
+    --from-file=ssh-privatekey=<path/to/.ssh/id_rsa>
```

Comment:
Your change LGTM. I noticed that the syntax for the user-supplied value of the path to the SSH key file is not in accordance to our [contribution guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output). Below is a suggested change if you are up for it. Thank you!

```suggestion 
    --from-file=ssh-privatekey=<path_to_ssh_key_file>
```

Reply:
Hello @sjhala-ccs,
   Thanks so much for looking at this issue, I appreciate it. I think in this case I will  keep it "as is" for now, as it matches the current version and I would like to keep it consistent. I will make a note of this in making these suggested changes as time permits. Appreciate it!